boofuzz/sessions.py

```diff
@@ -365,7 +365,8 @@ def __init__(self, session_filename=None, index_start=1, index_end=None, sleep_t
         self._skip_current_element_after_current_test_case = False
         self._post_test_case_methods = []
 
-        self.web_interface_thread = self.build_webapp_thread(port=self.web_port)
+        if self.web_port is not None:
+            self.web_interface_thread = self.build_webapp_thread(port=self.web_port)
```

Comment:
Did you run into a case where self.web_port is None? Seems like it should generally be set to something. Was the intention to allow one to set web_port=None to disable the web server?

Reply:
Correct. The intention is to allow a user to disable the web interface.